python/hail/context.py

```diff
@@ -114,11 +114,43 @@ def init(sc=None, app_name="Hail", master=None, local='local[*]',
              log='hail.log', quiet=False, append=False,
              min_block_size=1, branching_factor=50, tmp_dir='/tmp',
              default_reference="GRCh37"):
-    HailContext(sc, app_name, master, local, log, quiet, append, min_block_size, branching_factor, tmp_dir, default_reference)
+    """Initialize Hail and Spark.
+
+    Parameters
+    ----------
+    sc : pyspark.SparkContext, optional
```

Comment:
Your use of `optional` seems inconsistent, and I forget what convention we decided on.